README.md

```diff
@@ -1483,7 +1484,156 @@ Borrow from:
 https://github.com/dwyl/phoenix-todo-list-tutorial#10-clear-completed
 
 
-## 11. Deploy to Heroku
+## 11. Live Components
+
+LiveView provides the Live Components feature to group UI state and events.
+In this section we're going to see how to use component for items.
+
+The first step if to create a new file `lib/live_view_todo_web/live/item_component.ex`:
+
+```elixir
+defmodule LiveViewTodoWeb.ItemComponent do
+  use LiveViewTodoWeb, :live_component
+  alias LiveViewTodo.Item
+
+  attr(:items, :list, default: [])
+
+  def render(assigns) do
```

Comment:
_Reeeaaaallly_ not a fan of this `~H"""` templates inside the `render` function ... 😕 
is this the _only_ way of doing LiveComponents ...? 🤷‍♂️ 
This just feels like they've borrowed the _worst_ parts (from a maintainability perspective)
from React-land ...
can we not render partials and keep everything in `.heex` template files? 💭

Reply:
Indeed Phoenix has been inspired a lot by React, https://fly.io/blog/love-letter-react/

I've started by using [colocating templates](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-colocating-templates), however I think the `attr` and `slot` (see https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-attributes) only work with using the `~H` sigil

Reply:
Hmm ... yes, I'd forgotten about that blog post ... 🙄 

```elixir
defmodule DemoWeb.CounterLive do
  use Phoenix.LiveView

  def render(assigns) do
    ~H"""
    <div>
      <p>You clicked <%= @count %> times</p>
      <button phx-click="inc">Click me</button>
    </div>
    """
  end

  def mount(_, _, socket), do: {:ok, assign(socket, count: 0)}

  def handle_event("inc"), do: {:noreply, update(socket, :count, &(&1 + 1))}
end
```
There's so much "magic" going on in this one file it's _suuuuuper_ unfriendly to beginners. 😢